src/components/rapl/linux-rapl.c

```diff
@@ -405,6 +405,9 @@ _rapl_init_component( int cidx )
 
 		case 42:	/* SandyBridge */
 		case 58:	/* IvyBridge */
+		case 183:	/* RaptorLake-S/HX */
+      case 186:	/* RaptorLake */
```

Comment:
Hello @willowec can you align these case's and rebase? 

I will approve and merge after that.

Reply:
Done. I'll make sure to change my editor to use tabs for indentation instead of spaces. Thank you!

Reply:
I think some files use tabs while others use spaces.  At least that is what I have noticed when making changes throughout the library. Thank you for the changes!